packages/generator/src/sdk-metadata/code-sample-util.spec.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  VdmActionImport,
+  VdmFunctionImport,
+  VdmParameter,
+  VdmServiceMetadata
+} from '../vdm-types';
+import {
+  getActionFunctionImport,
+  getActionFunctionParams,
+  getFunctionWithMinParameters,
+  getFunctionWithoutParameters,
+  getLevensteinClosestFunction,
+  getODataEntity
+} from './code-sample-util';
+
+describe('code-sample-utils entity', () => {
+  it('gets entity based on levenshtein algorithm', () => {
+    const service = {
+      originalFileName: 'DummyClass',
+      entities: [{ className: 'DummyCollection' }, { className: 'DummyClass' }]
+    } as VdmServiceMetadata;
+    expect(getODataEntity(service.originalFileName, service.entities)).toEqual(
+      service.entities[1]
```

Comment:
Here I would prefer to see the actual object because it is faster to grep that index 1-> which is in the array.